mono/mini/mini-arm.c

```diff
@@ -24,6 +24,10 @@
 #include <mono/utils/mono-threads-coop.h>
 #include <mono/utils/unlocked.h>
 
+#ifndef DISABLE_INTERPRETER
```

Comment:
Please remove this `#ifndef DISABLE_INTERPRETER` otherwise it's going to break the android build where it's actually disabled, due to missing symbol definition of `MonoInterpFrameHandle`